pkg/agent/baker.go

```diff
@@ -953,6 +953,9 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"GetPrivateEgressProxyAddress": func() string {
 			return config.ContainerService.Properties.SecurityProfile.GetProxyAddress()
 		},
+		"ArtifactStreamingEnabled": func() bool {
```

Comment:
we should add a UT in baker_test.go?

Reply:
Will be adding it after the behavior change in the next PR. If `ArtifactStreamingEnabled` is true, then it should have a particular containerd config. If not true, then the standard one.